rsocket-transport-netty/src/main/java/io/rsocket/transport/netty/WebsocketUriHandler.java

```diff
@@ -33,7 +36,7 @@
  */
 public final class WebsocketUriHandler implements UriHandler {
 
-  private static final List<String> SCHEME = Arrays.asList("ws", "wss");
+  private static final List<String> SCHEME = Arrays.asList("ws", "wss", "http", "https");
```

Comment:
If http and https are registered here, what is the HTTP/2 scheme?

Reply:
I'm happy to change, in rsocket-cli to e.g. h2.  But would rather discuss why websockets should be https first.

Reply:
Yep, perfectly reasonable question.  I wasn’t sure about it, but I wanted to bring it into alignment with the [`WebsocketClientTransport` code](https://github.com/rsocket/rsocket-java/blob/db47113f214d1f916244cf0a1dca7fbc7ae0deee/rsocket-transport-netty/src/main/java/io/rsocket/transport/netty/client/WebsocketClientTransport.java#L77).  It doesn’t feel great to me (I was thinking HTTP/2 as well), so I’d be game to trimming that and the `UriUtils` back to `ws(s)` only.